slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/ChatDeleteScheduledMessageParamsIF.java

```diff
@@ -0,0 +1,25 @@
+package com.hubspot.slack.client.methods.params.chat;
+
+import org.immutables.value.Value.Immutable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.methods.interceptor.HasChannel;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ChatDeleteScheduledMessageParamsIF extends HasChannel {
+  @Override
+  @JsonProperty("channel")
+  String getChannelId();
+
+  String getScheduledMessageId();
+
+  @JsonProperty("as_user")
```

Comment:
Do we need this? Looks like [the argument](https://api.slack.com/methods/chat.deleteScheduledMessage#arg_as_user) looks the same.

Reply:
Yes, it's fully the same, I'll update and fix it. Thank you.